extension/manifest.json

```diff
@@ -6,7 +6,7 @@
     {
       "matches": ["*://*/*projects*/issues/*",
                   "*://*/*browse/*",
-                  "*://*/*secure/*projectKey=*"
+                  "*://*/*secure/*"
```

Comment:
for anyone looking at that code this would look quite suspicious.

Can't we have something else?
having `/browse/` and `issues` seems to cover most, adding `/RapidBoard*` I cannot find any other URL pattern that wouldn't match.

Reply:
How would you change it exactly?